### PR TITLE
Attempt to pass ASF website checks

### PR DESCRIPTION
**Pull request by devinsba**
The ASF has a set of website checks that our site must pass. I have taken a quick and dirty pass at making the site pass them and maintain some semblance of nice looking.

The report is available at: https://whimsy.apache.org/pods/project/zipkin

Here is a look at the changes that I made. I pretty much copied verbiage from the SkyWalking website.

Screenshot

### CODE REVIEW

**Comment by wu-sheng** [comment]
👍

File: public/css/zipkin.css [hunk 1]
```diff
@@ -51,3 +51,11 @@ table.wide-table td {
...
   font-size: 0.8em;
 }
+
+p.disclaimer {
+  font-size: 0.8em;
+}
+
+p.notice {
+  font-size: 0.8em;
+}
```
Comment (huydx) [inline public/css/zipkin.css:61]: nits: new line at the end

Comment (shakuzen) [reply public/css/zipkin.css:61]: We can update the `.editorconfig` file to enforce this (as long as people are using editors that understand and apply editorconfig rules). We can do it in a separate PR, though.
https://github.com/apache/incubator-zipkin-website/blob/master/.editorconfig

**Review by abesto — APPROVED** [review]
Looks good, thanks for taking the time!

**Review by shakuzen — APPROVED** [review]

**Comment by shakuzen** [comment]
Does merging this mean we'll stop merging things into https://github.com/openzipkin/openzipkin.github.io because they will get out of sync? Or are we syncing both ways?

**Comment by codefromthecrypt** [comment]
guess I wasnt watching this. clutch says the following need to be in place which maybe this change addresses 

ASF Foundation link is missing form website

ASF Events link is missing from website

Apache License link is missing from website

Apache Thanks link is missing from website

Apache Security link is missing from website

Apache Sponsorship link is missing from website

Trademarks are missing from website

Disclaimer is missing from website

**Comment by devinsba** [comment]
Yup, those are the checks this is intended to address

**Review by codefromthecrypt — APPROVED** [review]

**Pull request merged**